apps/platform/src/app/components/projects/ProjectListMap.tsx

```diff
@@ -102,15 +140,8 @@ export default function ProjectListMap() {
 						disableDefaultUI: false,
 						styles: mapStyles,
 					}}
+					onLoad={(map: google.maps.Map) => (mapRef.current = map)}
```

Comment:
@kkatusic you don't get the error from Verel in dev? Have you tried building to replicate?

I think this should work:
```suggestion
					onLoad={
					  (map: google.maps.Map) => {
					    mapRef.current = map;
					  }
					}
```

Reply:
@b0rza , thx, after investigation I see that we don't even need additional package, main package already has that option :), but needed to move map loading script to parent component because after you change tab two times loading error occurs